src/charm.py

```diff
@@ -49,36 +50,26 @@ def __init__(self, *args):
         self._container_name = self._service_name = "smf"
         self._container = self.unit.get_container(self._container_name)
 
-        # NRF relation endpoint
-        self._nrf_requires = NRFRequires(charm=self, relationship_name="nrf")
+        self._nrf_requires = NRFRequires(charm=self, relation_name="fiveg_nrf")
 
-        # Databases libraries
-        self._default_database = DatabaseRequires(
-            self, relation_name="default-database", database_name=DATABASE_NAME
+        self._database = DatabaseRequires(
+            self, relation_name="database", database_name=DATABASE_NAME
```

Comment:
What is the difference between `database` and `smf-database`?

Reply:
That'd be for accessing different collections Mongo collections, I think it should be clear from my question in the daily, I wasn't totally sure as well.

Reply:
I think `database` should probably have a clearer name and we can figure out what it is used for.